src/Renderer/ThreeExtended/Feature2Mesh.js

```diff
@@ -202,10 +202,14 @@ function coordinateToLines(coordinates, properties, options) {
     return new THREE.LineSegments(geometry);
 }
 
-function coordinateToPolygon(coordinates, properties, options) {
+function coordinateToPolygon(coordinates, properties, options = {}) {
     const indices = [];
     const vertices = new Float32Array(3 * coordinates.coordinates.length);
     const colors = new Uint8Array(3 * coordinates.coordinates.length);
+    let batchId;
+    if (options.buildBatchId) {
+        batchId = new Uint16Array(2 * coordinates.coordinates.length);
```

Comment:
You only need 1 value per vertices in the non-extruded case.

Reply:
And I wouldn't mind always building the `batchId` property, instead of using using an opt-in option.

Reply:
> You only need 1 value per vertices in the non-extruded case.

oops

> And I wouldn't mind always building the batchId property, instead of using using an opt-in option.

ok